### DISCUSSION TITLE
View Linked repositories from project?

### DISCUSSION THREAD

**Discussion by oliverKimdu** [community/community#56923] [Repositories]
### Select Topic Area

General

### Body

Is it possible to see which repos are linked to a project? It seems that there is no way to jump from a project to an associated repo, only the other direction.

**Comment by aahouzi** [comment]
Within GitHub, it is not possible to directly view which repositories are linked to a project from the project page. The project page primarily serves as a high-level overview of tasks and issues organized within the project.

However, you can navigate from a repository to the associated project. On a repository's main page, you can click on the "Projects" tab to see any projects associated with that repository. This provides a way to jump from the repository to the associated project.

Unfortunately, there is no direct counterpart on the project page to view linked repositories. To determine which repositories are linked to a project, you may need to manually check the individual repositories or use the GitHub API to retrieve this information programmatically.

You can use the GitHub REST API or the GitHub GraphQL API to fetch details about repositories and their associated projects. By making API calls, you can retrieve information about the repositories and their connections to projects.
